jekyll/_cci2/(WIP) api-v2-pipelines-parameters-scopes.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: classic-docs
+title: "API v2 Pipelines, Parameters and Scopes"
+short-title: "API v2 Miscellaneous"
+description: "detailed information about API v2 pipelines, parameters, and scopes"
+categories: [getting-started]
+order: 1
+---
+
+The CircleCI API v2 was designed to utilize pipelines, parameters, and scopes so you can work with API v2 more efficiently.
```

Comment:
The current phrasing makes it sounds like the entire function of the CircleCI API v2 is to utilize pipelines/parameters, but that's not quite true! API v2 is an entirely new interface to ACCESS those features. I'd suggest rephrasing to "The CircleCI API v2 introduces the ability to work with pipelines, pipeline parameters and workflows in your automation" .

I'm also not sure which feature `scopes` is referring to...maybe i missed a feature :P

Reply:
fixed